WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -511,7 +501,7 @@ function(task, colIdx) {
 		htmlArr[idx++] = this.getColorForStatus(task.status);
 		htmlArr[idx++] = "' style='width:"+ task.pComplete + "%;'></div></div>";
 	}
-    htmlArr[idx++] = "</td><td width=75 align=right><table><tr>";
+    htmlArr[idx++] = "</td><td class='IconOuterWrapper'><table class='IconInnerWrapper'><tr>";
```

Comment:
is there any use of IconOuterWrapper class here?
I am not able to find any css selector for this